libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -51,10 +57,21 @@ export class BookSearchComponent {
   }
 
   searchBooks() {
```

Comment:
You're still calling this function from `searchExample` - this will create another subscription for every call.

Reply:
Updated code.